rruntime/goroutine-factory.go

```diff
@@ -22,21 +26,30 @@ import (
 //    rruntime.Go(func() {
 //      	rt.workerProcess(worker)
 //    })
-func Go(function func()) {
+//When the go routine panics with an error, and if a custom errorHandler is passed
+func GoHandleError(function func(), errorHandler func(err error)) {
 	go func() {
 		ctx := bugsnag.StartSession(context.Background())
 		defer func() {
 			if r := recover(); r != nil {
+
```

Comment:
Also, are we planning to give support for suppressing panic with custom panic handler? If so, we should call custom panic handler first and also decide if we want to further here to notify bugsnag, reportapperror etc...